### ISSUE TITLE
About training loss

### ISSUE THREAD

**Issue by Ltohka** [indu1ge/DepthMaster#14]
I followed readme to reproduce the project and when I do stage 2 training, the loss contains `nan`. 
```python
if torch.isnan(depth_pred).any():
                    logging.warning("model_pred contains NaN.")
```
I put this code after [this line](https://github.com/indu1ge/DepthMaster/blob/main/src/trainer/trainer_s2.py#L219), and its output shows that this is a `nan`.

So Where did I go wrong?

<img width="772" height="845" alt="Image" src="https://github.com/user-attachments/assets/57714fc2-2e36-441b-9912-3c24724e36ea" />

**Comment by indu1ge** [comment]
Hi,

Regarding the nan loss issue, you can try commenting out Lines 325 and 326 in `train_s2.py`.

Additionally, we have fixed several other issues. Please refer to my recent commit: 63999f63553bbab4ca80207c9b47534253abfae3 (click the hash to view the changes).

After updating the code, the training should be normal, and the loss should be around 0.006.

Hope this helps!
